spec/features/ownership_transfer_spec.rb

```diff
@@ -20,88 +20,54 @@
   end
 
   before do
-    sign_in original_owner
-    go_to_dashboard_files
+    # sign_in original_owner
+    # go_to_dashboard_files
   end
 
-  describe 'When I request a file transfer:', :js do
-    context 'For a file I do not own' do
+  describe 'When I request a work transfer:', :js do
```

Comment:
Yes.  These are the tests for the dashboard UI where the user checks the work and then select transfer from a dropdown menu.  The menu had TODO transfer or something of the sort for the text.  So I figured I get to implementing that after the backend was functioning.